src/App.svelte

```diff
@@ -2,6 +2,14 @@
   import { exampleData } from "./ExampleData.js";
   export let servers = exampleData;
   console.log(exampleData);
+  export function grabip(text) {
```

Comment:
are we gonna do snake case camel case or no case

Reply:
Agreed we should be doing camelCase

Reply:
CamelCase for JS